frontend/src/components/assessments/data-exchange/AssessmentKeyUpload.vue

```diff
@@ -0,0 +1,530 @@
+<template>
+  <v-container class="mb-6" fluid>
+    <div class="borderless">
+      <v-row>
+        <v-col class="d-flex justify-center">
+          <h1>Upload Assessment Keys</h1>
+        </v-col>
+      </v-row>
+
+      <v-row no-gutters>
+        <v-col align-self="start" />
+        <v-col cols="2" offset="0" align-self="center">
+          <v-sheet class="pa-1 ma-1">
+            <v-autocomplete
+              id="Session"
+              v-model="selectedSessionID"
+              variant="underlined"
+              :items="sessionSearchNames"
+              label="Please select the session"
+              :clearable="true"
+              color="#003366"
+              item-title="title"
+              item-value="value"
+              autocomplete="off"
+            />
+          </v-sheet>
+        </v-col>
+        <v-col align-self="end" />
+      </v-row>
+      <v-row>
+        <v-col class="d-flex justify-center">
+          <span class="mr-1">
+            Please choose the session and click the link below to select your
```

Comment:
being VERY nitpicky here :) - I think change this to say "button" instead of link

Reply:
Great catch.